src/java.base/share/classes/java/security/KeyStore.java

```diff
@@ -1421,6 +1421,7 @@ public final void store(OutputStream stream, char[] password)
      *          algorithm could not be found
      * @throws    CertificateException if any of the certificates included in
      *          the keystore data could not be stored
+     * @throws    UnsupportedOperationException if this method is not supported
```

Comment:
I would change "method" to "operation" as it is a bit more consistent with the exception.

Reply:
Fixed as suggested. Thanks for the review.